treeherder/log_parser/parsers.py

```diff
@@ -319,6 +319,11 @@ def parse_line(self, line, lineno):
                 title, value = line.split("<br/>", 1)
                 artifact["title"] = title
                 artifact["value"] = value
+            # and split TinderboxPrint or similar long lines
+            elif "href" in line and "title" in line:
```

Comment:
Hey man, I have a fear that this may end up being a little brittle.  If the data comes in slightly different than we expect, these splits could fail.  Would you be up for trying this using https://docs.python.org/2/library/htmlparser.html ?  I think that would be a much nicer way to go.

Thanks!